training/distributed_training/pytorch/model_parallel/vit/submit_trainer_vit.ipynb

```diff
@@ -0,0 +1,566 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #13.        "fp16": False,
</pre><p>Let's set fp16 by default <code>True</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3572/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>

Reply:
In order fp16 work, we need changes in Rubik to be merged.